### PR TITLE
Check propTypes for tooltip, radio and radio group

### PR DESCRIPTION
**Pull request by dorokhovich-mikhail**
TICKET - [TPMTI-102](https://technekes.atlassian.net/browse/TPMTI-102)

### CODE REVIEW

**Comment by johnny-five** [comment]
I, @Dantiff, have checked:

### Style/Standards/Process

* [ ] If there is commented out code that should be deleted
* [ ] That the change implements what was requested in the [ticket]()
* [ ] For adequate test coverage for changed lines or critical code paths

### Code Quality

* [ ] Do the classes follow [CodeClimate 7 Patterns to Refactor](http://blog.codeclimate.com/blog/2012/10/17/7-ways-to-decompose-fat-activerecord-models/)
* [ ] If there are more idiomatic ways of using our frameworks/libraries
* [ ] Are there alternative implementations or refactors that increase readability/understandability and/or maintainability
* [ ] I can actually understand what the code is doing (if you're struggling to understand, then others may as well)
* [ ] This change is ready for production, if not should there should be a [feature flag](https://github.com/voormedia/flipflop)

### Security and Privacy

* [ ] Do the changes raise any security or privacy concerns

File: src/Tooltip/Tooltip.component.tsx [hunk 1]
```diff
@@ -21,6 +18,9 @@ export class Tooltip extends React.Component<Props> {
     arrow: true,
     theme: Themes.defaultTheme,
   };
+  /**
+   * What the reason of this function?
+   */
```
Comment (Dantiff) [inline src/Tooltip/Tooltip.component.tsx]: I don't think we should leave this here. Seems like its not being used. Git Lens says @ben-dalton had added it 4 months when setting up the component.

File: src/Tooltip/Tooltip.stories.tsx [hunk 2]
```diff
@@ -26,7 +25,7 @@ storiesOf('Tooltip', module).add(
         ],
         'top-start',
       )}
-      trigger="manual"
+      trigger="click"
```
Comment (Dantiff) [inline src/Tooltip/Tooltip.stories.tsx]: Tooltip arrow seems to have a different background color as compared to tooltip body

File: src/RadioButton/RadioButton.component.tsx [hunk 3]
```diff
@@ -138,7 +157,22 @@ export class RadioButton extends React.Component<Props> {
...
-    onChange: () => {},
+    id: uuid.v4(),
+    disabled: false,
+  };
+
+  state = {
+    checked: this.props.defaultChecked,
+  };
+
+  onChange = (event: any) => {
```
Comment (Dantiff) [inline src/RadioButton/RadioButton.component.tsx]: Console is complaining about this handler, could this be an issue when using the component in external apps?

image

File: src/Tooltip/Tooltip.stories.tsx (hunk 2 above)
Comment (dorokhovich-mikhail) [reply src/Tooltip/Tooltip.stories.tsx]: Fixed

File: src/Tooltip/Tooltip.component.tsx (hunk 1 above)
Comment (dorokhovich-mikhail) [reply src/Tooltip/Tooltip.component.tsx]: Removed

File: src/RadioButton/RadioButton.component.tsx (hunk 3 above)
Comment (dorokhovich-mikhail) [reply src/RadioButton/RadioButton.component.tsx]: There was confusing with checked and defaultChecked props.

Fixed

**Pull request merged**